src/Laravel/NorthstarUserProvider.php

```diff
@@ -67,8 +62,7 @@ public function retrieveByCredentials(array $credentials)
      */
     public function validateCredentials(Authenticatable $user, array $credentials)
     {
-        $token = $this->northstar->authorizeByPasswordGrant($credentials);
-
-        return ! is_null($token);
+        // Is this weird? Yes.
+        return true;
```

Comment:
I wrote up some details in the commit message (b218b43) and in #18. Short-answer is that `validateCredentials` is only ever called after `retrieveByCredentials` and there was no decent reason to make two separate HTTP requests here (or a good way to split up the process the way Laravel expects). I'm thinking about using moving away from using a user provider for that reason.